.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: ruby
 rvm:
   - 2.2.3
+before_script:
+  - sed -ie '/baseurl/s/\/m-lab\.github\.io//' _config.yml # sets baseurl to "" in _config for deploying w/o unnecessary url paths
```

Comment:
Absolutely, good catch.

This comment is now moved to the previous line and changed to suggested wording.

Also added the caret to the regex as suggested to match only at the beginning of a string.